src/libraries/System.Net.Security/src/System/Net/Security/Pal.OSX/SafeDeleteSslContext.cs

```diff
@@ -19,6 +20,8 @@ internal sealed class SafeDeleteSslContext : SafeDeleteContext
         private Interop.AppleCrypto.SSLWriteFunc _writeCallback;
         private ArrayBuffer _inputBuffer = new ArrayBuffer(InitialBufferSize);
         private ArrayBuffer _outputBuffer = new ArrayBuffer(InitialBufferSize);
+        private int _pendingInput;
```

Comment:
Can you add comments about what the values mean? Looks like there are four states... 1, 2, 3, 4, is that right?

Reply:
Would private enum be better?  I can certainly add comment as minimum.

Reply:
Either an enum or `const int` values. The field itself will need to be an `int` since you can't do interlocked on enums.

Reply:
updated + some small logic changes.